imap_processing/hit/l1a/hit_l1a.py

```diff
@@ -50,29 +34,19 @@ def hit_l1a(packet_file: str, data_version: str) -> list[xr.Dataset]:
     processed_data : list[xarray.Dataset]
         List of Datasets of L1A processed data.
     """
-    # TODO add logging
-
-    # Unpack ccsds file
-    packet_definition = (
-        imap_module_directory / "hit/packet_definitions/hit_packet_definitions.xml"
-    )
-    datasets_by_apid = packet_file_to_datasets(
-        packet_file=packet_file,
-        xtce_packet_definition=packet_definition,
-        use_derived_value=False,
-    )
+    # Unpack ccsds file to xarray datasets
+    datasets_by_apid = get_datasets_by_apid(packet_file)
 
     # Create the attribute manager for this data level
-    attr_mgr = ImapCdfAttributes()
-    attr_mgr.add_instrument_global_attrs(instrument="hit")
-    attr_mgr.add_instrument_variable_attrs(instrument="hit", level="l1a")
-    attr_mgr.add_global_attribute("Data_version", data_version)
+    attr_mgr = get_attribute_manager(data_version, "l1a")
 
-    # Process science to l1a.
+    # Process l1a data products
     if HitAPID.HIT_HSKP in datasets_by_apid:
-        datasets_by_apid[HitAPID.HIT_HSKP] = process_housekeeping(
-            datasets_by_apid[HitAPID.HIT_HSKP], attr_mgr
+        logger.info("Creating HIT L1A housekeeping dataset")
+        datasets_by_apid[HitAPID.HIT_HSKP] = process_housekeeping_data(
+            datasets_by_apid[HitAPID.HIT_HSKP], attr_mgr, "imap_hit_l1a_hk"
         )
+
     if HitAPID.HIT_SCIENCE in datasets_by_apid:
         # TODO complete science data processing
         print("Skipping science data for now")
```

Comment:
Should this print statement be removed? It doesn't look like science data is being skipped